modules/metrics/src/main/java/org/apache/ignite/internal/metrics/exporters/jmx/JmxExporter.java

```diff
@@ -173,7 +164,7 @@ private void unregBean(ObjectName bean) {
         try {
             ManagementFactory.getPlatformMBeanServer().unregisterMBean(bean);
         } catch (JMException e) {
-            log.error("Failed to unregister MBean: " + bean, e);
+            log.error("Failed to unregister MBean [bean=" + bean + ", err=" + e.getMessage() + ']', e);
```

Comment:
you can use `IgniteLogger#error(java.lang.String, java.lang.Throwable, java.lang.Object...)` instead of concatenation